feature: 프로젝트 폴더 구조를 수정했습니다.

ch1517 opened the pull request:
폴더 구조는 아래와 같습니다.
- **apis** : 서버와 소통하는 API Request 작업을 수행하는 파일
- **assets** : images, fonts와 같은 정적 파일
- **components** : 재사용이 가능한 컴포넌트 파일
- **components/common** : 전역적으로 사용되는 컴포넌트 파일
- **stores** : 상태 관리 변수 관련 파일
- **types** : typescript type/interface 관련 파일
- **utils** : 중복 로직 함수들을 관리하는 파일

ch1517 committed:
feature: 프로젝트 폴더 구조를 수정했습니다.

폴더 구조는 아래와 같습니다.
- apis : 서버와 소통하는 API Request 작업을 수행하는 파일
- assets : images, fonts와 같은 정적 파일
- components : 재사용이 가능한 컴포넌트 파일
- components/common : 전역적으로 사용되는 컴포넌트 파일
- stores : 상태 관리 변수 관련 파일
- types : typescript type/interface 관련 파일
- utils : 중복 로직 함수들을 관리하는 파일

deokgoo commented:
보통 파일 잡을때 .gitkeep이라는 파일을 만듭니다 (공식은 아닙니다)
tmp -> .gitkeep

https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep

deokgoo commented:
static file 은 nextjs 스럽게 public으로 이동 하는게 어떨까요? (image, font 등)
font는 CDN 사용할 생각하고 있어서 지워주셔도 될 것 같습니다. 

https://nextjs.org/docs/basic-features/static-file-serving

deokgoo commented:
추가적으로 부탁드리고 싶은것은
components, util 파일 alias로 사용하면 편할 것 같습니다 

https://nextjs.org/docs/advanced-features/module-path-aliases

# AS IS
```javscript
import test from '../../util/hoge';
```

# TO BE
```javscript
import test from '@/util/hoge';
```

ch1517 committed:
code review 수정사항 반영

- .gitkeep 사용
- static 파일 구조 변경(&font 폴더 삭제)
- path alias 사용을 위한 tsconfig.json 수정
